support-frontend/app/actions/AsyncAuthenticatedBuilder.scala

```diff
@@ -17,31 +15,22 @@ import scala.util.{Failure, Success}
 // to authenticate the user data.
 // TODO: considering porting this to identity-play-auth.
 // scalastyle:on
-class AsyncAuthenticatedBuilder[U](
-    userinfo: RequestHeader => Future[U],
-    defaultParser: BodyParser[AnyContent],
-    onUnauthorized: RequestHeader => Result
```

Comment:
the actual changes in this file are removing the unnneeded `onUnauthorized`, and hard coding the type `U`.  The rest is just tidyup

Reply:
I'm a big fan of removing unnecessary generics 👏